hellogsm-entity/src/main/java/team/themoment/hellogsm/entity/domain/application/entity/admission/AdmissionInfo.java

```diff
@@ -44,7 +44,7 @@ public class AdmissionInfo {
 
 
     @Column(name = "applicant_brith", nullable = false)
-    private LocalDate applicantBrith;
+    private LocalDate applicantBirth;
```

Comment:
이 부분 오타 있네요.
아래처럼 변경해야 합니다.
```suggestion
    private LocalDate applicantBrith;
```